.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     commit-message:
       prefix: "cargo"
     open-pull-requests-limit: 10
```

Comment:
you should bump this as well, if it's only done monthly 10 won't be enough.  maybe 40-50?

Reply:
makes sense, i increase that to 50.